### PR TITLE
#168781741 chore(setup-redux):set redux for react app

### PR DESCRIPTION
**Pull request by swaibat**
#### What does this PR do?
- setup redux in react app
#### Description of Task to be completed?
- setup redux
- install redux and react-redux
- add basic redux code and structure
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168781741](https://www.pivotaltracker.com/story/show/168781741)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/Redux/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { combineReducers } from './node_modules/redux';
```
Comment (urbanishimwe) [inline src/Redux/reducers/index.js:1]: 🛑the path `./node_modules/redux` does not exist. just do this `import { combineReducers } from 'redux';`

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+import Provider from 'react-redux';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(allReducers);
+
+ReactDOM.render(
+	<Provider store={store}>
+		<App />
+	</Provider>,
+	document.getElementById('root')
```
Comment (urbanishimwe) [inline src/index.js]: it will be good if you configure `Provider` under `App/App.js` as we discussed on it

File: src/index.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
 import ReactDOM from 'react-dom';
 import App from './App/App.js';
 import React from 'react';
+import { createStore } from 'redux';
+import allReducers from './Redux/reducers';
+import Provider from 'react-redux';
```
Comment (urbanishimwe) [inline src/index.js]: you import `Provider` by using `destructuring` like:
```
import { Provider } from 'react-redux';
```

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,15 @@
 import ReactDOM from 'react-dom';
 import App from './App/App.js';
 import React from 'react';
+import { createStore } from 'redux';
```
Comment (urbanishimwe) [inline src/index.js]: for team convention, `createStore` should be called inside `redux/store.js`. reach out to me for more clarif..

**Review by urbanishimwe — CHANGES_REQUESTED** [review]
nice workflow.
However, try to consider those comments and try to add some codes in the store.js
you were expected to create the store too

**Review by urbanishimwe — APPROVED** [review]

**Review by urbanishimwe — APPROVED** [review]

**Review by Crispy-rw — APPROVED** [review]

**Pull request merged**